blog/_posts/2018-08-08-one-point-zero.md

```diff
@@ -0,0 +1,161 @@
+> for general programming as Python, as easy for statistics as R, as natural for string
+> processing as Perl, as powerful for linear algebra as Matlab, as good at gluing programs
+> together as the shell. Something that is dirt simple to learn, yet keeps the most serious
+> hackers happy. We want it interactive and we want it compiled.
+
+A vibrant and thriving community has grown up around this language, with people from all
+around the world iteratively refining and shaping Julia in pursuit of that goal. Over 700
+people have contributed to Julia itself and even more people have made thousands of amazing
+open source Julia packages. All told, we have built a language that is:
+
+* **Fast**: Julia was designed from the beginning for high performance. Julia programs
+  compile to efficient native code for multiple platforms via LLVM.
+* **General**: It uses multiple dispatch as a paradigm, making it easy to express many
+  object-oriented and functional programming patterns. The standard library provides
+  asynchronous I/O, process control, logging, profiling, a package manager, and more.
+* **Dynamic**: Julia is dynamically-typed, feels like a scripting language, and has good
+  support for interactive use.
+* **Technical**: It excels at numerical computing with a syntax that is great for math, many
+  supported numeric data types, and parallelism out of the box. Julia's multiple dispatch
+  is a natural fit for defining number and array-like data types.
+* **Optionally typed**: Julia has a rich language of descriptive data types, and type
+  declarations can be used to clarify and solidify programs.
+* **Composable**: Julia’s packages naturally work well together. Matrices of unit
+  quantities, or data table columns of currencies and colors, just work — and with good
+  performance.
+
+Try Julia by [downloading version 1.0 now](https://julialang.org/downloads/). If you’re
+upgrading code from Julia 0.6 or earlier, we encourage you to first use the transitional 0.7
+release, which includes deprecation warnings to help guide you through the upgrade process.
+Once your code is warning-free, you can change to 1.0 without any functional changes. The
+package ecosystem is in the midst of taking advantage of this stepping stone and many
+packages are already 1.0-compatible! This includes ...
+
+The single most significant new feature in Julia 1.0, of course, is a commitment to language
+API stability: code you write for Julia 1.0 will continue to work in Julia 1.1, 1.2, etc.
+The language is “fully baked.” The core language devs and community alike can focus on
+packages, tools, and new features built upon this solid foundation.
+
+But Julia 1.0 in not just about stability, it also introduces several new, powerful and
+innovative language features. Some of the new features since version 0.6 include:
+
+* A brand new built-in [package manager](https://docs.julialang.org/en/latest/stdlib/Pkg/)
```

Comment:
It'd be nice to use the `en/v1.0` permalink here.  We don't seem to have the 1.0 manual available yet, though. :-\

Reply:
We can just fix that up after the fact.